outgoing.go

```diff
@@ -54,21 +53,20 @@ func bounce(w http.ResponseWriter, sig, url string) {
 		return
 	}
 
-	safeUrl := html.EscapeString(url)
-	if safeUrl[:10] == "javascript" {
+	if url[:10] == "javascript" {
 		debug("URL starts with javascript url: %v", url)
 		errorResp(w)
 		return
 	}
-	if !strings.Contains(safeUrl, "://") {
-		safeUrl = "http://" + safeUrl
+	if !strings.Contains(url, "://") {
```

Comment:
Yeah I can't think of a use-case would be for links that don't contain `://`.  It would also be nice to restrict to just `http`/`https` too since there's probably not a good case for allowing other protocols.